libcontainerd/remote/client_unix.go

```diff
@@ -1,4 +1,6 @@
-package libcontainerd // import "github.com/docker/docker/libcontainerd"
+// +build !windows
```

Comment:
should this file be named `client_linux.go` instead? (it previously was linux-only, now it's "unix", so also built on, e.g. bsd)

Reply:
Oh. I hadn't noticed I switched `_linux.go` to `_unix.go`, wasn't intentional. Good catch. I'll revert that.

Reply:
(Out of interest, is BSD still built, or did that effort die out?)

Reply:
I think some people still maintain a fork, and sometimes I've seen a PR come by.

Definitely not a priority, but if we can prevent breaking them, that's always a win :-)